numba/cuda/tests/cudapy/test_cuda_array_interface.py

```diff
@@ -205,6 +205,21 @@ def test_negative_strided_copy_to_host(self):
         expected_msg = 'D->H copy not implemented for negative strides'
         self.assertIn(expected_msg, str(raises.exception))
 
+    def test_masked_array(self):
+        h_arr = np.random.random(10)
+        h_mask = np.random.randint(2, size=10, dtype='bool')
+        c_arr = cuda.to_device(h_arr)
+        c_mask = cuda.to_device(h_mask)
+
+        # Manually create a masked CUDA Array Interface dictionary
+        masked_cuda_array_interface = c_arr.__cuda_array_interface__.copy()
```

Comment:
I thought you can simply do
```python
masked_cuda_array_interface = c_arr.copy()
```
according to https://github.com/numba/numba/issues/4130#issuecomment-499906702:
> `mask` should be an object that exports the `__cuda_array_interface__`, rather than the dictionary itself.